.travis.yml

```diff
@@ -0,0 +1,20 @@
+sudo: false
+language: node_js
+cache:
+  directories:
+    - node_modules
+notifications:
+  email: true
+node_js:
+  - '8'
+before_install:
+  - npm install -g yarn@1.3.2
+install:
+  - npm install
```

Comment:
I ultimately gave up - locally `yarn` command did install `run-p` alias from `npm-run-all` but on CI I could not get it to work! `run-p` was always not installed. So I switched to `npm install` with cached `node_modules` for speed.